tests/Get/GetTest.php

```diff
@@ -1,7 +1,8 @@
 <?php
 use Gettext\Languages\Language;
+use PHPUnit\Framework\TestCase;
```

Comment:
Any reason to switch to namespaced PHPUnic class names? Since we are using PHPUnit 4.8, those namespaced classes are just an alias of the non-namespaced classes...

Reply:
since the PHPUnit 4.8, I think it should use the class-based to support stable PHPUnit version in the future,

Reply:
> since the PHPUnit 4.8, I think it should use the class-based to support stable PHPUnit version in the future,

We can do that when we'll drop support for PHP 5.x (which isn't a change I have in my mind in the foreseeable future).

Reply:
The class-based PHPUnit namespace also support the PHP 5.x.
I think it's not the problem.